its/ruling/src/test/resources/expected/php-S2014.json

```diff
@@ -4,5 +4,5 @@
 2679,
 2683,
 2685,
-]
+],
```

Comment:
@vilchik-elena The trailing comma is useless, but all other JSON files seem to have it. I don't know why it wasn't there